### PR TITLE
setup native_osx and unisson to work with the same docker image

### PR DESCRIPTION
**Pull request by coreoas**

### CODE REVIEW

**Comment by coreoas** [comment]
Do not merge right now, image version will have to be updated after this PR is merged:

https://github.com/EugenMayer/docker-image-unison/pull/15

File: lib/docker-sync/sync_strategy/native_osx.rb [hunk 1]
```diff
@@ -47,8 +47,8 @@ def start_container
         env = {}
         raise 'sync_user is no longer supported, since it is not needed. Use sync_userid only please' if @options.key?('sync_user')
 
-        ignore_strings = expand_ignore_strings
-        env['UNISON_EXCLUDES'] = ignore_strings.join(' ')
```
Comment (EugenMayer) [inline lib/docker-sync/sync_strategy/native_osx.rb]: env['UNISON_EXCLUDES'] = ignore_strings.join(' ') got lost i guesS?

Comment (EugenMayer) [reply lib/docker-sync/sync_strategy/native_osx.rb]: oh no i see, you just push them on

**Comment by EugenMayer** [comment]
that was about what i was doing, but a lot of good stuff already went in!

**Comment by EugenMayer** [comment]
Awesome contribution, thank you!

**Pull request merged**